From 45f97fd8898fdc17ea774d5715a3413adbde18f7 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Sat, 09 May 2020 10:47:13 +0300
Subject: [PATCH] [SW-0] goya2: enable sris mode in pcie controller

Add missing setting to enable SRIS mode. we've enabled the
PCIe phy setting but the PCIe controller ws missing.

Change-Id: Ice8734255f8e691b5d2d7cf94e4fc9f32db53709
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/common/pcie.c b/common/pcie.c
index a3c18e7..7daedf1 100644
--- a/common/pcie.c
+++ b/common/pcie.c
@@ -123,8 +123,10 @@
 		boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS_L);
 
 		if (boot_strap &
-		    PSOC_GLOBAL_CONF_BOOT_STRAP_PINS_L_SRIS_MODE_MASK)
+		    PSOC_GLOBAL_CONF_BOOT_STRAP_PINS_L_SRIS_MODE_MASK) {
 			regmap_writel(1, mmPCIE_PHY_PIPE_RX_SRIS_MODE_EN);
+			regmap_writel(1, mmPCIE_AUX_APP_SRIS_MODE);
+		}
 
 		pcie_wa_snps_case_01038891(boot_strap);
 		regmap_writel(1, mmPSOC_GLOBAL_CONF_PCIE_PHY_RST_N);
